setup.cfg

```diff
@@ -33,6 +33,8 @@ per-file-ignores =
     src/pip/_internal/utils/filesystem.py: B014
     src/pip/_internal/network/cache.py: B014
     src/pip/_internal/utils/misc.py: B014
+enable-extensions = G002, G003, G010, G201, G202
```

Comment:
@McSinyx Why aren't we enabling the entire "G" class of errors here?

Reply:
I don't remember the exact reason but I wasn't happy with the plugin's suggestion (the `extra` thing) and thus disabling those wanting such.  I'd be avoiding going through the reformatting twice, but I guess it doesn't hurt enabling it *now* since I'll shift my focus to GH-8435.